docs/en/operation/Upgrade.md

```diff
@@ -69,6 +69,29 @@ Alluxio 2.x makes significant changes in the RPC layer,
 so pre-2.0.0 clients do not work with post-2.0.0 servers, and vice-versa.
 Upgrade all applications to use the alluxio-2.x client.
 
+Please refer to the following steps：
+1. Back up to the default backup folder `/alluxio_backups` in the root under storage system. This default backup directory can be configured by setting `alluxio.master.backup.directory`.
```

Comment:
Better leave the details to a link to an existing section on this `backup` command:
https://docs.alluxio.io/ee/user/stable/en/operation/Admin-CLI.html#backup

```suggestion
1. Back up the metadata of the files in Alluxio. Please refer to the [documentation](https://docs.alluxio.io/ee/user/stable/en/operation/Admin-CLI.html#backup) on `backup` command.
```